packages/ejson-shell-parser/src/eval.ts

```diff
@@ -98,7 +103,7 @@ const memberExpression = (node: CallExpression, withNew: boolean): any => {
         throw new Error('Expected CallExpression property to be an identifier');
 
       const args = node.arguments.map((arg) => walk(arg));
-      return calleeThis[calleeFn].apply(calleeThis, args);
+      return calleeThis[calleeFn](...args);
```

Comment:
Just checking, none of these `calleeThis` functions use `this` in their own scope right?